### PR TITLE
Fix Local US & Update the Klowd

### PR DESCRIPTION
**Pull request by brokenstream**

### CODE REVIEW

**Comment by R4IN2007** [comment]
There's one in particular that doesn't have a Channel ID but sorta shares a channel in Canada?
called Game+
Name might be different in the US?

FNTSY Sports Network
https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/fntsy_720p/playlist.m3u8

**Comment by brokenstream** [comment]
> There's one in particular that doesn't have a Channel ID but sorta shares a channel in Canada? called Game+ Name might be different in the US?
> 
> FNTSY Sports Network https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/fntsy_720p/playlist.m3u8

Thanks for that. The channel list the website gives me doesn't have that one on it at all. Wikipedia says that FNTSY Sports rebranded to Game+ in 2019, so that appears to be right and Klowd just never changed the URL.

File: streams/us_klowdtv.m3u [hunk 1]
```diff
@@ -1,33 +1,123 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="",Action Hollywood Movies (720p) [Geo-blocked]
+https://actionhollywood-klowdtv.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="AlwaysFunnyVideos.us",Always Funny Videos (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live3/alwaysfunny_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="",Animal Cam (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:6]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 2]
```diff
@@ -1,33 +1,123 @@
...
+https://amg00442-zmg-amg00442c2-klowdtv-us-2123.playouts.now.amagi.tv/playlist/amg00442-zazoommediagroup-zmgwild-klowdtvus/playlist.m3u8
+#EXTINF:-1 tvg-id="",Aspire TV Life (720p)
+https://uptv-aspiretvlife-klowdtv.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="AWE.us",AWE (720p)
 https://cdn.klowdtv.net/803B48A/awe_aws_ms/AWE.m3u8
 #EXTINF:-1 tvg-id="AWE.us",AWE (720p)
 https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live1/awe_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="beINSPORTSXTRAenEspanol.us",BeIN SPORTS XTRA en Espanol (720p) [Geo-blocked]
+https://bein-esp-klowdtv.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Beond TV (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:16]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 3]
```diff
@@ -1,33 +1,123 @@
...
+https://amg01681-beondtv-amg01681c1-klowdtv-us-1826.playouts.now.amagi.tv/playlist/amg01681-beondtv-beondtv-klowdtvus/playlist.m3u8
+#EXTINF:-1 tvg-id="BratTV.us",Brat TV (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:18]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 4]
```diff
@@ -1,33 +1,123 @@
...
+https://amg00350-amg00350c1-klowdtv-us-2536.playouts.now.amagi.tv/playlist/amg00350-bratstudiofast-brat-klowdtvus/playlist.m3u8
+#EXTINF:-1 tvg-id="",Cine en Espanol (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:20]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 5]
```diff
@@ -1,33 +1,123 @@
...
 https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live1/cine_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="Circle.us",Circle [Geo-blocked]
 https://circle-klowdtv.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="DemandAfrica.us",Demand Africa (720p)
-https://demandafrica-klowdtv.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="FidoTV.us",Fido TV (720p) [Not 24/7]
-https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live3/fido_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="",Diya TV (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/diyatv_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="",Eartho TV (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:28]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 6]
```diff
@@ -1,33 +1,123 @@
...
+#EXTINF:-1 tvg-id="France24Espanol.fr",France 24 Espanol (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/france24sp_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="FunRoads.us",Fun Roads TV (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live3/funroads_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="GamePlus.ca",Game+ (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/fntsy_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="GameShowNetworkEast.us",Game Show Network East (720p) [Not 24/7]
 https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/gsn_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="GBNews.uk",GB News (720p)
+https://amg01076-lightning-amg01076c7-klowdtv-us-2128.playouts.now.amagi.tv/playlist/amg01076-lightning-gbnews-klowdtvus/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:49]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 7]
```diff
@@ -1,33 +1,123 @@
...
+https://naviofrequency-horrormachine-klowdtv.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="HSN.us",HSN (720p)
-https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live3/hsn_720p/playlist.m3u8
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/hsn_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="HuntChannel.us",Hunt Channel (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/huntchannel_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="InfoWars.us",InfoWars (720p)
+https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/infowars_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="",In Touch Plus (720p)
+https://amg01540-apexmedia-amg01540c2-klowdtv-us-1989.playouts.now.amagi.tv/playlist/amg01540-apexmediafast-intouchplus-klowdtvus/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:59]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 8]
```diff
@@ -1,33 +1,123 @@
...
+#EXTINF:-1 tvg-id="JovemPanNews.br",Jovem Pan News (720p)
+https://amg01391-addigital-jovempan-klowdtv-cpe4b.amagi.tv/playlist/amg01391-addigital-jovempan-klowdtv/playlist.m3u8
+#EXTINF:-1 tvg-id="",LATV (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:62]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 9]
```diff
@@ -1,33 +1,123 @@
...
+https://pursuitup-klowdtv.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Quietude 4k (720p) [Geo-blocked]
+https://naviofrequency-quietude4k-klowdtv.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="QVC.us",QVC Live (720p) [Not 24/7]
 https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live2/qvclive_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="RedseatTheFirst.us",Redseat The First (720p)
 https://redseat-thefirst-klowdtv.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Salem News Channel (720p) [Geo-blocked]
+https://amg00732-salemmediagroup-salemnews-klowdtv-79zyj.amagi.tv/playlist/amg00732-salemmediagroup-salemnews-klowdtv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Smart Healthy Green Living (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:92]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

File: streams/us_klowdtv.m3u [hunk 10]
```diff
@@ -1,33 +1,123 @@
...
+https://amg16109-shgliving-amg16109c1-klowdtv-us-2235.playouts.now.amagi.tv/playlist/amg16109-shglivingnetwork-smarthealthygreenliving-klowdtvus/playlist.m3u8
 #EXTINF:-1 tvg-id="SonyMovies.us",Sony Movie Channel (720p)
 https://cdn.klowdtv.net/803B48A/n1.klowdtv.net/live1/smc_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="SonyMovies.us",Sony Movies (Sony Movie Channel)
 http://n1.klowdtv.net/live1/smc_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="SportsGrid.us",SportsGrid (720p)
 https://sportsgrid-klowdtv.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="SportsNewsHighlights.us",Sports News Highlights (720p)
```
Comment (BellezaEmporium) [inline streams/us_klowdtv.m3u:100]: GEO

403 ERROR
The request could not be satisfied.
The Amazon CloudFront distribution is configured to block access from your country.

**Review by BellezaEmporium — COMMENTED** [review]
What's cool is that KlowdTV almost got all of the local channels that once were only available in MyBTV. They might go further, who knows. Adressed a few comments about geo-blocked streams. Might need reconfirmation from the Trace channels from another country, as Trace's a French company.

**Comment by brokenstream** [comment]
> What's cool is that KlowdTV almost got all of the local channels that once were only available in MyBTV. They might go further, who knows. Adressed a few comments about geo-blocked streams. Might need reconfirmation from the Trace channels from another country, as Trace's a French company.

I've tested Trace on Canada, Netherlands, France and as a user from the US and they all worked well. Hope that covers all the rest.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**